about.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles/styles.css" />
+  </head>
+
+<body class="about-body">
+  <nav class="top-right open nav-container">
+    <a class="disc l1" href="index.html">Home</a>
+     
+    <a class="disc l2" href="about.html">About</a>
+
+    <a class="disc l3" href="portfolio.html">Portfolio</a>
+    
+    <a class="disc l4" href="hobby-blog.html">Hobbies</a>
+  </nav>
```

Comment:
while it's not imperative, think about making your navigation links an unordered list in the future. It is a common practice that you'll see!